docs/visual-basic/programming-guide/language-features/linq/how-to-modify-data-in-a-database-by-using-linq.md

```diff
@@ -66,7 +66,7 @@ Language-Integrated Query (LINQ) queries make it easy to access database informa
     Private Sub Form1_Load(ByVal sender As System.Object,   
                            ByVal e As System.EventArgs  
                           ) Handles MyBase.Load  
-      db = New northwindDataContext()  
+      Dim db = New northwindDataContext()  
```

Comment:
The `Dim` here is incorrect; db is declared on line 64. Declaring it here causes it to not be in scope for the call in `RefreshData`.